src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+/* eslint-disable no-console */
+
+type State = {
+  time: string;
+};
+
+type Props = {
+  clockName: number | string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    time: new Date().toLocaleTimeString(),
```

Comment:
Almost always it is better to store raw data in the component state(redux store/database/localStorage etc.). Because in that you will have more possibilities how you can use stored data.